app/controllers/api/v1/bibs_controller.rb

```diff
@@ -28,4 +40,34 @@ def delete_teacher_sets
       ]
     }.to_json
   end
+
+  def request_valid?(method_name)
```

Comment:
Why not separate this validation method to 2 different validation methods? One for delete and the other for update/create.